.circleci/config.yml

```diff
@@ -148,6 +151,7 @@ workflows:
   build:
     jobs:
       - build-Docs
+      - cleanup-gke-jobs
```

Comment:
why extra here, it is already later?

Reply:
Removed that line since I already got the sample run that I wanted: https://app.circleci.com/pipelines/github/PyTorchLightning/pytorch-lightning/9518/workflows/dea2a6af-740e-4bd7-b828-cfd7ab37acb3/jobs/53637